### PR TITLE
Enums in SlevomatCodingStandard.Classes.ClassStructure

### PR DESCRIPTION
**Pull request by michnovka**
Was getting 

 1 | ERROR | An error occurred during processing; checking has been aborted. The error message was: You need configure all class groups. These groups are missing from your configuration: enum cases.

Without this in phpcs.xml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ public methods, protected methods, private methods,
 public final methods, public static final methods, protected final methods, protected static final methods,
 public abstract methods, public static abstract methods, protected abstract methods, protected static abstract methods,
 public static methods, protected static methods, private static methods,
-private methods
+private methods, enum cases
```
Comment (kukulich) [inline README.md:150]: Please add it after `uses` to new line.

**Review by kukulich — CHANGES_REQUESTED** [review]

**Comment by kukulich** [comment]
@michnovka Please squash it to one commit.

**Comment by michnovka** [comment]
@kukulich done

**Review by kukulich — APPROVED** [review]

**Comment by kukulich** [comment]
Thanks.

**Comment by michnovka** [comment]
Going thru the changelog, seems that another "shortcut" was added "readonly properties". This is not required tho, but maybe should be mentioned in the README.md as well?

**Pull request merged**